tests/test_types.py

```diff
@@ -836,3 +836,27 @@ class Model(BaseModel):
         assert exc_info.value.flatten_errors() == result
     else:
         assert Model(foo=value).foo == result
+
+
+def test_gt():
+    class Model(BaseModel):
+        a: conint(gt=-1) = 0
+
+    assert Model(a=0).dict() == {'a': 0}
+    with pytest.raises(ValidationError) as exc_info:
+        Model(a=-1)
+    assert (
+        'ensure this value is greater than -1 (type=value_error.number.gt; limit_value=-1)'
```

Comment:
why you want to use string representation of errors not `.flatten_errors()` like in all other tests?

Reply:
I was just doing the quickest thing, doing it various different ways in tests is a good thing.